app/.gitignore

```diff
@@ -1 +1,2 @@
 /build
+/src/main/AndroidManifest.xml
```

Comment:
You need this file to build the app, you shouldn't remove it from the repo. If it has sensitive information, you should move it to a properties file or another file that can be pulled out of the repo.